scripts/scons_plugin/pkgconfig.py

```diff
@@ -45,6 +45,32 @@ def write_file(target, source, env):
 
     return env.File(target)
 
+# Find and return path to the pkgconfig directory of the latest version of `lib_name`
+# library installed by brew.
+#
+# When `required` = True error is raised unless the search was successful. Otherwise in
+# case of any error None is returned.
+def FindBrewLibrary(env, lib_name, required=True):
+    if not env.Which('brew'):
+        if required:
+            env.Die('brew not found')
+        else:
+            return None
```

Comment:
We should not fail if the user does not have brew. Using brew is just a convenient shortcut, but user is free to install openssl by any other means.

We can pass required=False, but I think we can just remove required flag at all, for now.

Reply:
TODO: if brew is installed and contains openssl then append path to openssl's pkgconfig path to `env['PKG_CONFIG_PATH']`.

Reply:
fixed